### PR TITLE
added necessary and unique tests

### PR DESCRIPTION
**Pull request by hackerashish25**
added tests to file  test_utils.py

### CODE REVIEW

File: tests/test_utils.py [hunk 1]
```diff
@@ -15,23 +15,27 @@ def test_removeall_list():
     assert removeall(4, []) == []
     assert removeall(4, [1, 2, 3, 4]) == [1, 2, 3]
     assert removeall(4, [4, 1, 4, 2, 3, 4, 4]) == [1, 2, 3]
+    assert removeall(1,[2,3,4,5,6]) == [2,3,4,5,6]
```
Comment (MrDupin) [inline tests/test_utils.py]: After a comma, you should be adding a space. This is the case for all the additions in this PR.

Other than that, nice work!

Comment (hackerashish25) [reply tests/test_utils.py]: @MrDupin Thank You for your review

Comment (MrDupin) [reply tests/test_utils.py]: Remember to add spaces in all the additions you made.

Comment (hackerashish25) [reply tests/test_utils.py]: @MrDupin all test cases are perfectly added now

**Pull request merged**